CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 2.15.0-rc.0 / 2019-12-18
+
+* [BUGFIX] Rules: Prometheus no longer accepts badly written rules and alerts that have same metric name and labelset. #6469
```

Comment:
Let's not moralise, we now catch rules/alerts that produce duplicates due to templating.